src/libsync/syncfilestatustracker.cpp

```diff
@@ -157,6 +162,20 @@ SyncFileStatus SyncFileStatusTracker::fileStatus(const QString& systemFileName)
     return SyncFileStatus();
 }
 
+void SyncFileStatusTracker::slotPathTouched(const QString& systemFileName)
+{
+    QString fileName = systemFileName.normalized(QString::NormalizationForm_C);
+
+    QString folderPath = _syncEngine->localPath();
+    if (!fileName.startsWith(folderPath))
+        return;
+
+    QString localPath = fileName.mid(folderPath.size());
+    _dirtyPaths.insert(localPath);
+
+    emit fileStatusChanged(systemFileName, SyncFileStatus::StatusSync);
```

Comment:
`systemFileName`?    Are you sure this should not be `fileName`
At least that's what all the other stuff are doing.